src/group_theory/quotient_group.lean

```diff
@@ -266,6 +266,29 @@ def equiv_quotient_of_eq {M N : subgroup G} [M.normal] [N.normal] (h : M = N) :
   right_inv := λ x, x.induction_on' $ by { intro, refl },
   map_mul' := λ x y, by rw map_mul }
 
+/-- Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →* B / (B' ⊓ B)` induced by the inclusions. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →+ B / (B' ⊓ B)` induced by the inclusions."]
+def quotient_map_of_subgroup_of_le {A' A B' B : subgroup G}
```

Comment:
```suggestion
def quotient_map_subgroup_of_of_le {A' A B' B : subgroup G}
```
Since this produces a `quotient_map` between `subgroup_of`s given a proof `of` `le`